### DISCUSSION TITLE
CGGT%40UPenn/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#230] [Announcements]
# CGGT%40UPenn/

None

https://opencs.app/CGGT%40UPenn/

**Comment by Yangxinyee** [comment]
主要cv和robo做的多 这个可以申嘛  适合不？

**Reply by jichumao** [reply]
Animation会用到robo相关的只是, CV也会用到一些但是比较有限，任选课也可以选CV， 主要都是CG课

**Comment by Snackiller** [comment]
请问现在转MCIS或者dual MCIS的难度大吗

**Comment by jichumao** [comment]
认真学问题不大，Dual更合理一些


------------------ Original ------------------
From: Shuhao Gao ***@***.***&gt;
Date: Sat, Jul 20, 2024 1:31 PM
To: opencsapp/opencsapp.github.io ***@***.***&gt;
Cc: Jichu Mao ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [opencsapp/opencsapp.github.io] CGGT%40UPenn/ (Discussion #230)





 
请问现在转MCIS或者dual MCIS的难度大吗

**Reply by Snackiller** [reply]
学长可以问问你是怎么转专业或者dual成MCIS的吗
